spra-play-server/src/main/resources/application.conf

```diff
@@ -6,14 +6,18 @@ dataExplorer {
     # here you can define the tables that will be displayed in the data explorer with their
     # respective configuration
 
-    # users {
-      # tableName = "users"
-      # primaryKeyField = "user_id"
-      # hiddenColumns = ["password", "email"]
-      # nonEditableColumns = ["user_id", "email", "created_at", "verified_on", "name", "last_name"]
-      # canBeDeleted = false
-      # filterableColumns = ["name", "last_name"]
-    # }
+    users {
+      tableName = "users"
+      primaryKeyField = "user_id"
+      hiddenColumns = ["password", "email"]
+      nonEditableColumns = ["user_id", "email", "created_at", "verified_on", "name"]
+      canBeDeleted = false
+      filterableColumns = ["name", "last_name"]
+      createFilter {
+        requiredColumns = ["name", "email", "password"]
+        nonRequiredColumns = ["last_name", "discord_webhook_url"]
```

Comment:
do we really need both options? also, I see no tests.

Reply:
It's to make known React-Admin which fields will be displayed and required in the /table/create page. The required and non required fields will be defined in the database (with DEFAULT constraint)